lib/experimental/fonts/font-library/class-wp-font-family.php

```diff
@@ -493,7 +530,21 @@ private function create_font_post() {
 	private function update_font_post( $post ) {
 		$post_font_data = json_decode( $post->post_content, true );
 		$new_data       = WP_Font_Family_Utils::merge_fonts_data( $post_font_data, $this->data );
-		$this->data     = $new_data;
+		$intersecting   = $this->get_intersecting_font_faces( $post_font_data['fontFace'], $new_data['fontFace'] );
```

Comment:
Can we expect that the data will always contain a `fontFace` key, or should we be more defensive here?

Reply:
Nope, there can be font families without a fontFace array defined or an empty array.

Reply:
Thanks, done in [b07fdd0](https://github.com/WordPress/gutenberg/pull/54490/commits/b07fdd083687c250a396588cb9b525d6c3cb568c)